test/config-next/ra.json

```diff
@@ -151,7 +151,8 @@
 				},
 				"grpc.health.v1.Health": {
 					"clientNames": [
-						"health-checker.boulder"
+						"health-checker.boulder",
+						"consul.boulder"
```

Comment:
nit: the other lists of client names are in alphabetical order; this should list consul.boulder before health-checker.boulder